admin_guide/http_proxies.adoc

```diff
@@ -0,0 +1,140 @@
+use these proxies is as simple as setting standard environment variables in
+configuration or JSON files.
+
+== Configuring Hosts for Proxies
+
+Add the `*NO_PROXY*`, `*HTTP_PROXY*`, and `*HTTPS_PROXY*` environment variables
+to the *_/etc/sysconfig/openshift-master_* and *_/etc/sysconfig/openshift-node_*
+files:
+
+====
+----
+HTTP_PROXY=http://USERNAME:PASSWORD@10.0.1.1:8080/
+HTTPS_PROXY=https://USERNAME:PASSWORD@10.0.0.1:8080/
+NO_PROXY=master.hostname.example.com
+----
+====
+
+== Proxying Docker Pull
+
+. Edit the *_/etc/sysconfig/docker_* file and add the variables in shell format:
++
+====
+----
+HTTP_PROXY=http://USERNAME:PASSWORD@10.0.1.1:8080/
+HTTPS_PROXY=https://USERNAME:PASSWORD@10.0.0.1:8080/
+NO_PROXY=master.hostname.example.com
+----
+====
+
+. Restart the master service:
++
+----
+# systemctl restart openshift-master
+----
+
+. If you had already imported ImageStreams before creating the proxy
+configuration, delete, then re-run the process:
++
+----
+# oc delete imagestreams -n openshift --all
+# oc create -f image-streams.json -n openshift
+----
+
+== Importing Image Streams
+
+== S2I Builds
+
+S2I builds fetch dependencies from various locations as needed. Fork the desired
+codebase and create a file named *_.sti/environment_*. The contents of the file
+are simple shell variables. Most libraries will look for `*NO_PROXY*`,
+`*HTTP_PROXY*`, and `*HTTPS_PROXY*` variables and react accordingly:
+
+S2I builds fetch dependencies from various locations as needed. The process for
+setting this up is in the link:../dev_guide/builds.html#environment-files[builds
+topic]. The contents of the file are simple shell variables. In setting an
+instance for use with web proxies, insert the following into the
+*_.sti/environment_* file. When seeing the `*NO_PROXY*`, `*HTTP_PROXY*`, and
+`*HTTPS_PROXY*` variables, OpenShift will react accordingly:
+
+====
+----
+HTTP_PROXY=http://USERNAME:PASSWORD@10.0.1.1:8080/
+HTTPS_PROXY=https://USERNAME:PASSWORD@10.0.0.1:8080/
+NO_PROXY=master.hostname.example.com
+----
+====
+
+== Setting Environment Variables in Pods
+
+Set the following environment variables in a DeploymentConfig to pass database
+connection information. The same can be done for configuring a Pod's proxy at
+runtime:
+
+====
+----
+{
+  "apiVersion": "v1beta1",
+  "kind": "DeploymentConfig",
+  "metadata": {
+    "name": "frontend"
+  },
+  "template": {
+    "controllerTemplate": {
+      "podTemplate": {
+        "desiredState": {
+          "manifest": {
+            "containers": [
+              {
+                "env": [
+                  {
+                    "name": "HTTP_PROXY",
+                    "value": "http://USER:PASSWORD@IPADDR:PORT"
+                  },
+----
+====
+
+== Git Repository Access
+
+
+
+include::dev_guide/builds.html[tag=using-a-proxy-for-git-cloning]
+
+////
+To use GitHub behind a proxy, set an environment variable on the stiStrategy:
```

Comment:
As above, this para is commented out. 

I can delete this section actually. Doesn't really serve a purpose anymore.

Aaaand deleted.